File: wrappers/dotnet/indy-sdk-dotnet/AnonCredsApi/CredDefAlreadyExistsException.cs
namespace Hyperledger.Indy.AnonCredsApi
{
    /// <summary>
    /// Exception thrown when attempting create a credential definition that already exists.
    /// </summary>
    public class CredentialDefinitionAlreadyExistsException : IndyException
    {
        const string message = "The specified credential definition already exists.";

        /// <summary>
        /// Initializes a new CredDefAlreadyExistsException.
        /// </summary>
        internal CredentialDefinitionAlreadyExistsException() : base(message, (int)ErrorCode.AnoncredsCredDefAlreadyExistsError)
        {

        }
    }

}

File: wrappers/android/src/main/java/org/hyperledger/indy/sdk/anoncreds/CredDefAlreadyExistsException.java
package org.hyperledger.indy.sdk.anoncreds;

import org.hyperledger.indy.sdk.ErrorCode;
import org.hyperledger.indy.sdk.IndyException;

/**
 * Exception thrown when an anoncreds accumulator is full.
 */
public class CredDefAlreadyExistsException extends IndyException
{
	private static final long serialVersionUID = -6792822612990030627L;
	private final static String message = "The anoncreds accumulator is full.";

	/**
	 * Initializes a new CredDefAlreadyExistsException.
	 */
	public CredDefAlreadyExistsException() {
		super(message, ErrorCode.AnoncredsCredDefAlreadyExistsError.value());
	}
}